frontend/src/pages/Admin/components/CsvImport.js

```diff
@@ -253,7 +261,7 @@ export default function CsvImport(
                   </ul>
                 </div>
               )}
-              <Button className="margin-top-2" type="button" onClick={importCsvFile}>
+              <Button className="margin-top-2" type="button" onClick={importCsvFile} disabled={error}>
```

Comment:
I'd rather we didn't disable this, it's annoying since you have to refresh to resubmit, even if you change the file, plus the color contrast is poor.

Reply:
My concern here is they can still upload the bad file. Is there another way we can prevent this?

When you say "refresh to submit", all I had to do is change the selected file to enable the button.

Reply:
I guess we can just do this check once when they click the upload button instead.